### ISSUE TITLE
Multiple sign-in emails being sent

### ISSUE THREAD

**Issue by tyliec** [CodeWithAloha/HIERR#205]
**Describe the bug**

During the sign-in process, the following screen is presented to the user:

![Image](https://github.com/CodeWithAloha/HIERR/assets/15609358/de2ca489-f3d8-4f8b-8c5b-c1069df012a2)

If an email is inputted and the "Sign in with Email" button is pressed more than once in quick time frame, multiple sign-in emails are sent.

![Image](https://github.com/CodeWithAloha/HIERR/assets/15609358/a0bfcbf7-0f5e-4380-a1ce-1c114b03816f)

**Expected behavior**

Multiple emails cannot be sent from this screen.

**Comment by avenmia** [comment]
I don't think we have control over the email double send at the moment. It can be fixed if we redo that sign in page, but the risk of breaking sign in is too high this late. The page exists in node_modules/next-auth/src/core/pages/sign-in.tsx. You can customize your own sign-in page but have to handle some of the callbacks and redirects yourself which is a pain.  This could be addressed if #126 is fixed
